qchem/tests/test_molecular_hamiltonian.py

```diff
@@ -6,20 +6,47 @@
 
 from pennylane.vqe import Hamiltonian
 
-ref_dir = os.path.join(os.path.dirname(os.path.realpath(__file__)), "test_ref_files")
+import numpy as np
+
+
+symbols = ["C", "C", "N", "H", "H", "H", "H", "H"]
+coordinates = np.array(
```

Comment:
Can this be given in one line?

Reply:
I wrote it in one line by "black" changed it to a column :-)